lib/core/network/api/aws_amplify/amplify_service.dart

```diff
@@ -22,18 +22,18 @@ class AmplifyService {
   final EOSService eosService;
   final NetworkingManager networkingManager;
 
-  CognitoUserPool get userPool => CognitoUserPool(
-        remoteConfigService.userPoolId,
-        remoteConfigService.clientId,
+  CognitoUserPool userPool(Network network) => CognitoUserPool(
+        remoteConfigService.userPoolId(network),
+        remoteConfigService.clientId(network),
       );
   CognitoUser? cognitoUser;
   CognitoUserSession? session;
 
   RemoteConfigService get remoteConfigService => eosService.remoteConfigService;
```

Comment:
I guess this should be a single network specific object with all the ppp parameters in it.